public/app/dashboard/profileController.js

```diff
@@ -133,7 +133,7 @@
             'color': $scope.color.ACTIVITY_TIMELINE_RED
           },
           {
-            'name': ' Creations',
+            'name': 'Recreates',
```

Comment:
@paulkerschen I notice space removed above and space preserved before "Views/Likes".  Intentional?